src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useContext } from 'react';
+import { TodoFilter } from '../TodoFilter';
+import { ErrorMessage } from '../../types/ErrorMessages';
+import { TodoContext } from '../../context';
+import * as TodoClient from '../../api/todos';
```

Comment:
```suggestion
import * as todoClient from '../../api/todos';
```
It's not Component, Class (Constructor-function) or Type